stockroom/repository.py

```diff
@@ -12,35 +85,38 @@ def init(name, email, overwrite):
     repo = Repository(Path.cwd(), exists=False)
     if repo.initialized and (not overwrite):
         commit_hash = repo.log(return_contents=True)['head']
-        print(f'Repo already exists at: {repo.path}')
+        print(f'Hangar Repo already exists at {repo.path}. '
+              f'Initializing it as stock repository')
     else:
+        if not all([name, email]):
+            raise ValueError("Both ``name`` and ``email`` has to be non-empty"
+                             " strings for initializing hangar repository")
```

Comment:
this isn't checking if `name` and `email` are strings, it's just checking if the variables are not `Falsey`. These could be set to almost any `int` (except `0`), `float`, `bytes`, class instance, or any number of other `Truthy` values and still pass with errors down the line.

Reply:
My bad about the error message. What I needed to check was just that they are not `None` and let hangar raises any other errors. Made the changes